coralproject/talk
# New Queue

## New Queue

## What does this PR do?
- Separate Queue, called New (shows if there is no Pre-Mod queue)
- "New" queue only contains comments that have not been Accepted or Rejected

`Pre-mod` = stream/system set to pre-mod on

`Reported` = user reported, suspect words, `system reported` aka trust, perspective toxicity threshold, etc.

--  If someone hits a trust threshold -- is their comment marked as `reported` `flagged` or `premod`

So the options would be:
New | Reported | Approved | Rejected | All

or

Pre-Mod | Reported | Approved | Rejected | All

## How do I test this PR?
- Create a new asset
- Switch it to premod
- Go to admin
- Switch between articles 
- Notice that Pre-Mod Tab shows up when the article has premod enabled, and New when it's disabled.



- Try to go to `http://localhost:3000/admin/moderate/new/a36b75ed-e1d0-4c8b-8905-7c8e08971730`
- Erase the `new`
- It should go to `reported` if there is reported comments, then `premod` if premod is enabled, and if not, it will default to `new`
